### PR TITLE
Update/db schema

### PR DESCRIPTION
**Pull request by ccali11**
@shanejearley - The primary purpose of this branch / PR is to make our PG schema more flexible in terms of handling the permutations (combinations?) of users and addresses (accounts as well) from a web3 perspective (e.g., previously, our schema was preventing two separate users from sharing an address in their list of accounts AND it was preventing a user from switching their primary address to another address). To achieve this, a simple PG generated/managed `id` is used on both the `users` and `accounts` table, and a third composite table is used to associate `users` with `accounts`.

There are probably ways to optimize the `postgres.ts` and `schema.ts` scripts, but focused on getting it working and figured we can settle on the general direction first before optimizations.

We can defer to async review, but let me know if you'd like to review sync at any point.

### CODE REVIEW

File: common/data/src/schemas/users_accounts.schema.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "$id": "https://casimir.co/users_accounts.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "User_Account",
```
Comment (shanejearley) [inline common/data/src/schemas/users_accounts.schema.json:4]: @ccali11 change to "User Account"

File: common/data/src/providers/schema.ts [hunk 2]
```diff
@@ -80,14 +83,18 @@ export class Schema {
...
-            
+
             return column
         })
 
+        /** Check for composite key property and add the primary key if so */
+        if (compositeKey) columns.push(`PRIMARY KEY (${compositeKey})`)
+
         /** Make table name plural of schema objects (todo: check edge-cases) */
         const tableName = this.getTitle().toLowerCase() + 's'
```
[the remark this answers is not in the document]
Comment (shanejearley) [reply common/data/src/providers/schema.ts:94]: @ccali11 wrap with snakeCase helper and remove 's'

**Comment by ccali11** [comment]
@shanejearley - Need approval to merge this.

**Review by DemogorGod — APPROVED** [review]
Approved apon request

**Pull request merged**